static/variables/ui-variables.less

```diff
@@ -82,4 +82,4 @@
 
 // Other
 
-@font-family: '.SFNSText-Regular', 'SF UI Text', 'Lucida Grande', 'Segoe UI', Ubuntu, Cantarell, sans-serif;
+@font-family: 'system';
```

Comment:
@kuon Good point! This whole thing will need to be redone anyway - I'm gonna close this out for now till I have a better solution.